CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 5.1.0 (xxxx-xx-xx)
 **SDK**
 - **Update:** Added `allEmailList` field in `MAContact` to support multiple emails of a specific contact.
+- **Feature:** New `MiniAppMessageDelegate` interface method `func closeMiniApp(withConfirmation: , completionHandler:)` to handle the close miniapp action from MinApps.
```

Comment:
```suggestion
- **Feature:** Added a new optional interface `func closeMiniApp(withConfirmation: , completionHandler:)` in  `MiniAppMessageDelegate` to support the close mini-app action from MinApps.
```

Reply:
Thank you I have updated in the latest commit.